node/sc/kas/anchor_test.go

```diff
@@ -45,26 +45,26 @@ func testAnchorData() *types.AnchoringDataInternalType0 {
 		ParentHash:    common.HexToHash("2"),
 		ReceiptHash:   common.HexToHash("3"),
 		StateRootHash: common.HexToHash("4"),
-		BlockNumber:   big.NewInt(5),
+		BlockNumber:   big.NewInt(10),
 		BlockCount:    big.NewInt(6),
 		TxCount:       big.NewInt(7),
 	}
 }
 
-func _TestExampleSendRequest(t *testing.T) {
-	url := "http://anchor-anchor-api.dev.klaytn.com/v1/anchor"
-	xkrn := "krn:1001:anchor:test:operator-pool:rp1"
-	user := ""
-	pwd := ""
+func TestExampleSendRequest(t *testing.T) {
+	url := "http://anchor-api.dev.klaytn.com/v1/anchor"
+	xChainId := "1001"
+	user := "Basic"
+	pwd := "S0FTSzVQREtBUEtYUUw0MEE3NzlSSkRPOnJHVlBVd3lXMjNVYkMzUEFORUxoQkJoaDgwYjFCQ2dBVWxwQW5KSlc"
```

Comment:
Is that a real account of dev environment? The authentication information should not be revealed in public.

Reply:
Oops It's my mistakes. I will remote pwd.

Reply:
Please, rebase your commit so that the pwd is not logged in history.